source/sass/mixins/_spacing.scss

```diff
@@ -132,3 +132,24 @@
   list-style: none;
   @include padding(0, inline-start);
 }
+
+@mixin inset-inline($start: null, $end: null) {
```

Comment:
This and `insert-block` should ultimately use the `logical-property` mixin in a similar way to the `padding` mixin.